### DISCUSSION TITLE
bigfix/10.0/patch/Patch/Patch_SUSE/c_introduction

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#40] [Q&A]
# bigfix/10.0/patch/Patch/Patch_SUSE/c_introduction

BigFix Patch for SUSE Linux Enterprise provides an automated, simplified patching process that is administered from a single console. It gives you unified, near real-time visibility and enforcement to deploy and manage patches to all SUSE Linux endpoints.

https://help.hcltechsw.com/bigfix/10.0/patch/Patch/Patch_SUSE/c_introduction.html

**Comment by sweekumari** [comment]
suse12
